### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cvega**
I made some changes to the readme for other developers, this changes the way people run and develop locally.

### CODE REVIEW

**Comment by cvega** [comment]
I dont like these changes at all

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # test feature
-test repo
+test repo - new new features
```
Comment (michaelsainz) [inline README.md:2]: I don't like this feature, how about this -

`test repo - this is a much better feature`

**Review by michaelsainz — COMMENTED** [review]
This is another comment

File: README.md (hunk 1 above)
Comment (cvega) [reply README.md:2]: I like that, thanks for the suggestions

Comment (michaelsainz) [inline README.md:2]: On second thought, this works just fine.

**Pull request merged**